Language review for SRT over QUIC draft (#99)
Haivision/srt-rfc: draft-sharabayko-srt-over-quic.md

--- abstract

This document presents an approach to tunnelling SRT live streams over QUIC datagrams.

QUIC {{RFC9000}} is a UDP-based transport protocol providing TLS encryption, stream multiplexing,
and connection migration. It was designed to become a faster alternative to the TCP protocol {{RFC7323}}.

An Unreliable Datagram Extension to QUIC {{QUIC-DATAGRAM}} adds support for sending and receiving
unreliable datagrams over a QUIC connection.

SRT {{SRTRFC}} is a UDP-based transport protocol. Essentially, it can operate over any unreliable datagram transport.
SRT in live streaming configuration provides an end-to-end latency-aware mechanisms for packet loss recovery.

> If SRT fails to recover a packet lost within a specified latency, then the packet is dropped to avoid

Reviewer: ```suggestion
If SRT fails to recover a packet loss within a specified latency, then the packet is dropped to avoid
```

Recover packet loss until the latency restriction permits. Not a packet lost within the latency.

# Acknowledgments
{:numbered="false"}

It is worth acknowledging the participation of the following people in the project discussions: Ying Yin (Google), Ian Swett (Google), Victor Vasiliev (Google), Kazuko Oku (Fastly), Marc Cymontkowski (Haivision), Jake Weissman (Facebook), Jordi Cenzano (Facebook), Alan Frindell (Facebook), Jeongseok Kim (SK Telecom), Joonwoong Kim (SK Telecom).

> Quicly library {{QUICLY}} by Fastly was chosen to provide a QUIC datagram transport layer for SRT over QUIC PoC. We would like to thank Kazuho Oku (Fastly) for his help with answering questions and resolving some of the library issues quickly.

Reviewer: ```suggestion
Quicly library {{QUICLY}} from Fastly was chosen to provide a QUIC datagram transport layer for SRT over QUIC PoC. We would like to thank Kazuho Oku (Fastly) for his help.
```
